echopype/calibrate/cal_params.py

```diff
@@ -312,7 +312,19 @@ def get_vend_cal_params_power(beam: xr.Dataset, vend: xr.Dataset, param: str) ->
 
     # Select corresponding index and clean up the original nan elements
     da_param = da_param.sel(pulse_length_bin=idxmin, drop=True)
-    return da_param.where(~transmit_isnull, np.nan)  # set the nan elements back to nan
+
+    # Set the nan elements back to nan.
+    # Doing the `.where` will result in float64,
+    # which is fine since we're dealing with nan
+    da_param = da_param.where(~transmit_isnull, np.nan)
```

Comment:
Calling `.where` in this line will always result in `nan`. An outside discussion suggest that there need to be a case for `range_sample` that should output int:
> yes I think it is reasonable to default to float, the only exception would be something like our range_sample  values, since it is sample count

I don't think this should be addressed in this PR, and probably further discussion is needed to figure out how to address it.

Reply:
I think this is fine, unless computationally this becomes super slow. 
Basically the `nan` here will end up being `nan` in the output Sv after calibrating the values in `backscatter_r`.

Reply:
Yea, I'll have to revisit this since this was already in the code, I didn't wanna change it much and break something else for now 😛